### ISSUE TITLE
Microsoft Code Signing Certificate expired 12-02-2021 

### ISSUE THREAD

**Issue by MN-Joe** [PowerShell/vscode-powershell#3733]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md) guide.
- [X] I am sure this issue is with the _extension itself_ and does not reproduce in a standalone [PowerShell](https://github.com/PowerShell/PowerShell/issues/new/choose) instance.
- [X] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.

### Summary

![image](https://user-images.githubusercontent.com/71799135/146842978-54af720d-5857-49b0-9088-ce02e374265e.png)

### PowerShell Version

```console
Name                           Value                                                                                                                                                                                                
----                           -----                                                                                                                                                                                                
PSVersion                      5.1.19041.1320                                                                                                                                                                                       
PSEdition                      Desktop                                                                                                                                                                                              
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                              
BuildVersion                   10.0.19041.1320                                                                                                                                                                                      
CLRVersion                     4.0.30319.42000                                                                                                                                                                                      
WSManStackVersion              3.0                                                                                                                                                                                                  
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                  
SerializationVersion           1.1.0.1
```

### Visual Studio Code Version

```console
1.63.1
fe719cd3e5825bf14e14182fddeb88ee8daf044f
x64
```

### Extension Version

```console
ms-vscode.powershell@2021.10.2
```

### Steps to Reproduce

Load Script to get certificate to load and select always to add it to the trusted publisher but the cert is expired and the script refuses to run.

**Comment by MN-Joe** [comment]
Got word to try the Preview which was updated and I'm able to get that to work.

**Comment by andschwa** [comment]
@MN-Joe Ah! Confirmed here, ok great. Thanks!

**Comment by andschwa** [comment]
@MN-Joe I just released v2021.12.0, can you confirm that it resolves this issue?
